src/plugins/organisation/PMRWindow/src/pmrwindowplugin.cpp

```diff
@@ -47,7 +47,7 @@ PLUGININFO_FUNC PMRWindowPluginInfo()
     descriptions.insert("fr", QString::fromUtf8("une extension pour accéder <a href=\"https://models.physiomeproject.org/\">PMR</a>."));
 
     return new PluginInfo("Organisation", true, false,
-                          QStringList() << "Core" << "libgit2" << "zlib",
```

Comment:
```PMRSupport``` should be listed as a dependency for ```PMRWindow``` otherwise the plugins window won't work as expected.